orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarter.groovy

```diff
@@ -88,6 +91,14 @@ class DependentPipelineStarter implements ApplicationContextAware {
     def t1 = new Thread(new Runnable() {
       @Override
       public void run() {
+        if (user) {
+
+          def securityCtx = SecurityContextHolder.createEmptyContext()
+          // Using RunAsUserToken is an arbitrary Authentication objection - it's merely named the best for this use case.
```

Comment:
Why not [PreAuthenticatedAuthenticationToken](http://docs.spring.io/autorepo/docs/spring-security/4.1.2.RELEASE/apidocs/org/springframework/security/web/authentication/preauth/PreAuthenticatedAuthenticationToken.html) to get rid of the random key?